doc/source/yt4differences.rst

```diff
@@ -93,6 +93,47 @@ density = arbitrary_grid[('gas', 'density')]
 print(density)
 ```
 
+Off-Axis Projection for SPH Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The current `off_axis_projection` function will now support SPH projection plots.
+
+The following code generates an off-axis projection of an example dataset with a given
+normal vector, resolution, center, width, and smoothing quantitiy. 
+
+.. code-block:: python
+
+    import yt
+
+    ds = yt.load('GadgetDiskGalaxy/snapshot_200.hdf5')
+
+    normal_vector = [0., 1., 0.]
+
+    resolution = (256, 256)
+
+    left_edge = ds.domain_left_edge
+
+    right_edge = ds.domain_right_edge
+
+    center = [(left_edge[0] + right_edge[0])/2,
+              (left_edge[1] + right_edge[1])/2,
+              (left_edge[2] + right_edge[2])/2]
+
+    width = [(right_edge[0] - left_edge[0]),
+             (right_edge[1] - left_edge[1]),
+             (right_edge[2] - left_edge[2])]  
```

Comment:
One suggestion for this example: how about zooming on the galaxy similar to the last plot [here](https://yt-project.org/docs/dev/cookbook/gadget_notebook.html)? The density peak could be located simply by `_, center = ds.find_max('density')`. And `width` could be set similar to the galaxy size. It might be more visually appealing.

Reply:
The new example should be better.